### PR TITLE
PCQ-729 Add Backend and NodeJS kustomise config

### PR DESCRIPTION
**Pull request by davejones74**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PCQ-729

### Change description ###
Update to change pipeline to use kustomise in Frontend and Backend applications for non-prod environments.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/namespaces/pcq/pcq-backend/demo.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: pcq-backend
+spec:
```
Comment (adusumillipraveen) [inline k8s/namespaces/pcq/pcq-backend/demo.yaml:6]: You dont need to add thes files if they are not needed.

Comment (davejones74) [reply k8s/namespaces/pcq/pcq-backend/demo.yaml:6]: I thought as much, ok I'll remove. One of them has an environment value which we'll need to keep. Thanks.

File: k8s/namespaces/pcq/pcq-backend/pcq-backend.yaml [hunk 2]
```diff
@@ -3,15 +3,13 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: pcq-backend
-  namespace: pcq
   annotations:
-    fluxcd.io/automated: "true"
-    fluxcd.io/tag.java: glob:prod-*
+    hmcts.github.com/global-defaults: enabled
+    hmcts.github.com/prod-automated: enabled
```
Comment (adusumillipraveen) [inline k8s/namespaces/pcq/pcq-backend/pcq-backend.yaml:8]:
```suggestion
  
```

File: k8s/namespaces/pcq/pcq-backend/pcq-backend.yaml [hunk 3]
```diff
@@ -3,15 +3,13 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: pcq-backend
-  namespace: pcq
   annotations:
```
Comment (adusumillipraveen) [inline k8s/namespaces/pcq/pcq-backend/pcq-backend.yaml:6]:
```suggestion
```

File: k8s/namespaces/pcq/pcq-backend/pcq-backend.yaml [hunk 4]
```diff
@@ -3,15 +3,13 @@ apiVersion: helm.fluxcd.io/v1
...
-    fluxcd.io/automated: "true"
-    fluxcd.io/tag.java: glob:prod-*
+    hmcts.github.com/global-defaults: enabled
+    hmcts.github.com/prod-automated: enabled
 spec:
   releaseName: pcq-backend
   rollback:
     enable: true
```
Comment (adusumillipraveen) [inline k8s/namespaces/pcq/pcq-backend/pcq-backend.yaml:12]:
```suggestion
```

File: k8s/namespaces/pcq/pcq-frontend/pcq-frontend.yaml [hunk 5]
```diff
@@ -3,10 +3,9 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: pcq-frontend
-  namespace: pcq
   annotations:
```
Comment (adusumillipraveen) [inline k8s/namespaces/pcq/pcq-frontend/pcq-frontend.yaml:6]:
```suggestion
```

File: k8s/namespaces/pcq/pcq-frontend/pcq-frontend.yaml [hunk 6]
```diff
@@ -3,10 +3,9 @@ apiVersion: helm.fluxcd.io/v1
...
-    flux.weave.works/automated: "true"
-    flux.weave.works/tag.nodejs: glob:prod-*
+    hmcts.github.com/global-defaults: enabled
+    hmcts.github.com/prod-automated: enabled
```
Comment (adusumillipraveen) [inline k8s/namespaces/pcq/pcq-frontend/pcq-frontend.yaml:8]:
```suggestion
```

**Review by adusumillipraveen — COMMENTED** [review]
minor comments

**Review by adusumillipraveen — APPROVED** [review]

**Pull request merged**